.github/workflows/ci.yml

```diff
@@ -11,39 +11,48 @@ env:
   GO_VERSION: ^1.21
   GOLANGCI_LINT_VERSION: v1.54.0
 
+permissions:
+  contents: read
+
 jobs:
   go_install:
     if: github.repository == 'Antonboom/testifylint'
     runs-on: ubuntu-latest
     steps:
-      - uses: rlespinasse/github-slug-action@v4.4.1
-      - uses: actions/setup-go@v4
+      - uses: rlespinasse/github-slug-action@102b1a064a9b145e56556e22b18b19c624538d94 # v4.4.1
```

Comment:
I don't like hashes and interested only in major version if we cannot write `@v4.4.1`
is it possible to configure `dependabot` to watch only for major updates?

Reply:
Having hashes is recommended by OSSF best practices concerning dependencies including github-actions. 
It's of course possible to configure dependabot to watch only major updates

Reply:
what is profit from it in specific context of this project?

> It's of course possible to configure dependabot to watch only major updates

please do it or close PR since I don't see a threat model here.
looks like change for the sake of change